cli/__snapshots__/cli_spec.js

```diff
@@ -1,55 +1,33 @@
-exports['cli --version no binary version 1'] = `
```

Comment:
any idea why these all got re-arranged? I think snap-shot-it should be sorting them in this version

I can't tell what changed as is

Reply:
any tips on getting them in the right order? I tried deleting `__snapshots__/cli_spec` and regenerating, but the diff is still meaningless compared to `develop`